redaxo/src/addons/media_manager/lib/effects/effect_convert2img.php

```diff
@@ -59,7 +59,7 @@ public function execute()
             return;
         }
 
-        if (!in_array(strtolower($ext), self::CONVERT_TYPES)) {
+        if (!in_array(strtolower($ext), array_keys(self::$convertTypes)) && !in_array(rex_file::mimeType($fromPath), self::$convertTypes)) {
```

Comment:
Statt in_array mit array_keys, besser einfaches isset()

```suggestion
        if (!isset(self::$convertTypes[strtolower($ext)]) && !in_array(rex_file::mimeType($fromPath), self::$convertTypes)) {
```